<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buy</class>
 <widget class="QWidget" name="Buy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="goods_num">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>140</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="import_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定采购</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_main">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="goods_total">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>采购公司</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品单价</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>140</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>采购数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>总 计</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="goods_company">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="goods_name">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="goods_price">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="goods">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>571</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="history">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>采购记录</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
